.github/ionic-issue-bot.yml

```diff
@@ -0,0 +1,79 @@
+triage:
+  label: triage
+  removeLabelWhenProjectAssigned: true
+  dryRun: false
+
+comment:
+  labels:
+    - label: "help wanted"
+      message: >
+        This issue has been labeled as `help wanted`. This label is added to issues
+        that we believe would be good for contributors.
+
+
+        If you'd like to work on this issue, please comment here letting us know that
+        you would like to submit a pull request for it. This helps us to keep track of
+        the pull request and make sure there isn't duplicated effort.
+
+        Thank you!
+    - label: "ionitron: needs reproduction"
```

Comment:
Does this label / action conflict with the one below? https://github.com/ionic-team/stencil-ds-output-targets/pull/240/files#diff-5ab3559a040d16f7b7db847060a448dddde18e8320e13b8d5737c31672708375R66

The message associated with this action seems to imply the issue will stay open (so long as they provide a repro), whereas the one below closes the issue. Am I reading this right?

Reply:
👍 Makes sense to me - thanks! (serious brain fog day 🙃)